anaUltraScurve.py

```diff
@@ -94,6 +95,10 @@
     myT.Branch( 'ROBstr', ROBstr, 'ROBstr/I' )
     mask = array( 'i', [ 0 ] )
     myT.Branch( 'mask', mask, 'mask/I' )
+    maskDead = array( 'i', [ 0 ] )
+    myT.Branch( 'maskDead', maskDead, 'maskDead/I' )
+    maskOutlier = array( 'i', [ 0 ] )
+    myT.Branch( 'maskOutlier', maskOutlier, 'maskOutlier/I' )
```

Comment:
If this is the goal then we should change how this is implemented.  These branch names are not revealing.  Instead you should have:

`mask`
`maskReason`

And then `maskReason` stores an enum style mask code, for example:

-1 -> not masked
0 -> Hot channel
1 -> Dead channel

etc...

Additionally what's the difference between `maskOutlier` and `mask` in the code they appear to be identical.